### PR TITLE
first couple of utf8 validation tests

### PR DESCRIPTION
**Pull request by weissi**
goes together with https://github.com/milseman/swift/pull/30

### CODE REVIEW

**Comment by weissi** [comment]
@milseman ok, deleted the code we don't need. This is ready for your first pass review.

File: Tests/UTF8StringTests/UTF8ValidationTests.swift [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        } catch {
+            XCTFail("not valid: \(error)", file: file, line: line)
+        }
+    }
+
+    func assertInvalidUTF8(_ bytes: UnsafeBufferPointer<UInt8>, expectedError: UTF8ValidationResult, _ message: Swift.String = "", file: StaticString = #file, line: UInt = #line) {
+        func errorFormat(_ value: (Swift.String, UTF8ValidationResult?)) -> (Swift.String, Swift.String) {
+            return (value.0, value.1.map(String.init(describing:)) ?? "<no error>")
+        }
+        func format<T>(_ values: [(Swift.String, T)]) -> Swift.String {
```
Comment (milseman) [inline Tests/UTF8StringTests/UTF8ValidationTests.swift]: What is this function used for?

**Review by milseman — APPROVED** [review]
Otherwise LGTM

**Comment by weissi** [comment]
@milseman updated

**Pull request merged**